src/steps/dump_and_push_on_s3.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const moment = require('moment');
+
+const { createBackup } = require('./backup-restore/index');
+const logger = require('../logger');
+
+function createS3(configuration) {
+  return new AWS.S3({
+    accessKeyId: configuration.S3_ACCESS_KEY,
+    secretAccessKey: configuration.S3_SECRET_ACCESS_KEY,
+    endpoint: configuration.S3_ENDPOINT,
+    region: configuration.S3_REGION,
+    // eslint-disable-next-line no-process-env
+    sslEnabled: !(process.env.NODE_ENV === 'test'),
```

Comment:
Alimenter cette variable avec false dans le fichier `src/config/extract-configuration-from-environment.js` si `NODE_ENV=test` comme sur le monorepo
https://github.com/1024pix/pix/blob/dev/api/lib/config.js#L341

```suggestion
    // eslint-disable-next-line no-process-env
    sslEnabled: configuration.s3Backup.sslEnabled
```

Reply:
modifié